shared/templates/static/oval/partition_for_var_log_audit.xml

```diff
@@ -4,6 +4,8 @@
       <title>Ensure /var/log/audit Located On Separate Partition</title>
       <affected family="unix">
         <platform>multi_platform_oval</platform>
+        <platform>multi_platform_debian</platform>
+        <platform>multi_platform_ubuntu</platform>
```

Comment:
Don't think that these are needed with `<platform>multi_platform_oval</platform>`